js/sign/src/cli.ts

```diff
@@ -27,13 +27,32 @@ function readOptions() {
     .parse(process.argv);
 }
 
+// Parses either an unencrypted or encrypted private key. For unencrypted keys,
+// reads the passphrase to decrypt an encrypted private key from either
+// `WEB_BUNDLE_SIGNING_PASSPHRASE` env var or if not set, it prompts passphrase
+// from the user.
 async function parseMaybeEncryptedKey(
   privateKeyFile: Buffer
 ): Promise<KeyObject> {
+  // If the env var is provided, that's most probably the right one so let's
+  // check this before the unencrypted case.
+  if (process.env.WEB_BUNDLE_SIGNING_PASSPHRASE !== '') {
```

Comment:
This seems like it wouldn't work - if the environment variable is not set at all, IIUC it would be `undefined`, which is `!== ''`. So I think you'd either have to make the condition `!=`, or explicitly catch falsy values, like so: `process.env.WEB_BUNDLE_SIGNING_PASSPHRASE && process.env.WEB_BUNDLE_SIGNING_PASSPHRASE !== ''`.